MBBSEmu.Tests/CPU/FISTP_Tests.cs

```diff
@@ -0,0 +1,93 @@
+﻿using System;
+using Iced.Intel;
+using Xunit;
+using static Iced.Intel.AssemblerRegisters;
+
+namespace MBBSEmu.Tests.CPU
+{
+    public class FISTP_Tests : CpuTestBase
```

Comment:
useful to add tests with different rounding modes as specified by the RC field in the FPU control word?

Reply:
Right now it's defaulting to rounding away from Zero. We can change it to respect the RC field eventually